index.html

```diff
@@ -482,55 +485,46 @@ <h2 class="heading">What is JHipster?</h2>
                         <i class="fa fa-4x">=</i>
                         <a href="/"><img id="hipster-head" src="images/logo/jhipster_family_member_0_head.svg" class="project-icon img-responsive"></a>
                     </div>
-                    <p class="lead">
-                        JHipster is a development platform to generate, develop and deploy
-                        <a class="dark" href="http://projects.spring.io/spring-boot/" target="_blank" rel="noopener">Spring Boot</a> + <a class="dark" href="https://angular.io/" target="_blank" rel="noopener">Angular</a> / <a class="dark" href="https://reactjs.org/" target="_blank" rel="noopener">React</a> / <a class="dark" href="https://vuejs.org/" target="_blank" rel="noopener">Vue</a>
-                        Web applications and Spring microservices. <br>
+                    <p class="lead font-weight-normal">
+                        JHipster is a development platform to quickly generate, develop, and deploy modern web applications and microservice architectures. We support many frontend technologies, including Angular, React, and Vue. We even have mobile app support for Ionic and React Native! On the backend, we support Spring Boot (with Java or Kotlin), Micronaut, Quarkus, Node.js, and .NET. For deployment, we embrace cloud native principles with Docker and Kubernetes. Deployment support exists for AWS, Azure, Cloud Foundry, Google Cloud Platform, Heroku, and OpenShift.
                     </p>
                 </span>
             </div>
             <div class="row">
                 <div class="col-md-6">
                     <h3 class="hipster-font heading">Goal</h3>
                     <p>
-                        Our goal is to generate for you a complete and modern Web app or microservice architecture, unifying:
+                        Our goal is to generate a complete and modern web app or microservice architecture, unifying:
                     </p>
                     <ul>
-                        <li>A high-performance and robust Java stack on the server side with Spring Boot</li>
-                        <li>A sleek, modern, mobile-first front-end with Angular, React and Bootstrap</li>
-                        <li>A robust microservice architecture with the JHipster Registry, Netflix OSS, the Elastic Stack and Docker</li>
-                        <li>A powerful workflow to build your application with Yeoman, Webpack and Maven/Gradle</li>
+                        <li>A high-performance and robust server-side stack with excellent test coverage</li>
+                        <li>A sleek, modern, mobile-first UI with Angular, React, or Vue + Bootstrap for CSS</li>
```

Comment:
nit: ```suggestion
                        <li>A sleek, modern, mobile-first UI with Angular, React, or Vue + Bootstrap CSS framework</li>
```

Reply:
I'm not sure we should add framework since technically Angular, React, and Vue are all frameworks.